components/composed/model/ModelColumns/ThumbnailColumn.tsx

```diff
@@ -17,28 +17,45 @@ export interface Thumbnail {
   image?: Image | null;
 }
 
-export interface ThumbnailNode extends Node {
+type ThumbnailColumn<T extends Node> = PartialColumnish<T> & {
   thumbnail?: Thumbnail | null;
-}
-const ThumbnailColumn = <NodeType extends ThumbnailNode>(
-  props: PartialColumnish<NodeType> = {}
-): Column<NodeType> => {
+  route?: string;
+};
+
+const ThumbnailColumn = <NodeType extends Node>({
+  route,
+  ...props
+}: ThumbnailColumn<NodeType>): Column<NodeType> => {
   const { t } = useTranslation();
 
   return {
     Header: <span className="a-hidden">{t("columns.thumbnail")}</span>,
     id: "thumbnail",
-    accessor: (row: NodeType) => row?.thumbnail?.image,
+    accessor: (thumbnail: NodeType) => thumbnail?.image,
```

Comment:
These lines are breaking the image - if you change it back to `(row: NodeType) => row?.thumbnail?.image,`, the images will appear again. The accessor passes in a node (`row`) and we need to get the thumbnail object from that row.